pkg/app.go

```diff
@@ -89,7 +89,7 @@
 		fetchersPkg.NewUnbondsFetcher(logger, appConfig, rpcs, tracer),
 		fetchersPkg.NewSigningInfoFetcher(logger, appConfig, rpcs, tracer),
 		fetchersPkg.NewRewardsFetcher(logger, appConfig, rpcs, tracer),
-		fetchersPkg.NewBalanceFetcher(logger, appConfig, rpcs, tracer),
+		fetchersPkg.NewBalanceFetcher(logger, appConfig.Chains, rpcs, tracer),
```

Comment:
Actually it's not about initiaizing with multiple chains - it's about passing only chains instead of the whole config as before, to make it easier to mock. But I updated it nevertheless, can you check?